.pre-commit-config.yaml

```diff
@@ -22,10 +22,11 @@ repos:
         args:
         - --rcfile=.pylintrc
         - --reports=no
+        exclude: ^tests/resources/.*_todo.*$
 -   repo: local
     hooks:
     -   id: py.test
         name: py.test
         language: system
-        entry: sh -c py.test
-        files: ''
\ No newline at end of file
+        entry: sh -c 'pip install -e .[testing] && python -m pytest'
```

Comment:
I wanted to make it nice so that you do not have to have python instaled or run the pre-commit in your specially-crafted virtualenv. I actually learned in the meantime that there is the "additional_dependencies" entry you can add in the python pre-commit and I will modify it to use it. This way you won't have to run 'git comit' in your virtualenv with dev_requirements.txt installed.

Reply:
I will submit it shortly - let me know if you will be ok with it :)

Reply:
Sure, I'm ok with that !